apps/gateway/src/routes/v1/chat/chat.handler.ts

```diff
@@ -0,0 +1,45 @@
+import { LogSources } from '@latitude-data/core/browser'
+import { streamToGenerator } from '@latitude-data/core/lib/streamToGenerator'
+import { addMessages } from '@latitude-data/core/services/documentLogs/index'
+import { captureException } from '@sentry/node'
+import { streamSSE } from 'hono/streaming'
+
+import { AppRouteHandler } from '$/openApi/types'
+import { ChatRoute } from '$/routes/v1/chat/chat.route'
+import { chainEventPresenter } from '$/common/documents/getData'
+
+// @ts-expect-error: streamSSE has type issues
+export const chatHandler: AppRouteHandler<ChatRoute> = async (c) => {
```

Comment:
For some reason types are not working at this level. cc @neoxelox

Reply:
mmm that's probably why I did what I did, handlers + route def altogether... this types are magical... but if `c.req.valid` does complain when getting unknown parameters from the request schema... I guess I am okay with that